R/cv_varsel.R

```diff
@@ -1328,6 +1376,18 @@ kfold_varsel <- function(refmodel, method, nterms_max, ndraws, nclusters,
     if (!requireNamespace("doRNG", quietly = TRUE)) {
       stop("Please install the 'doRNG' package.")
     }
+    if (verbose &&
+          requireNamespace("progressr", quietly = TRUE) &&
+          requireNamespace("progress", quietly = TRUE) &&
+          interactive()) {
+      use_progressr <- TRUE
+      progressr::handlers(global = TRUE)
+      progressr::handlers("progress")
+      p <- progressr::progressor(along = seq_along(inds))
+    } else {
+      use_progressr <- FALSE
+    }
+    .select <- .select
```

Comment:
Why is `.select <- .select` necessary?

Reply:
As .select is defined out of scope, parallel computation did not see it even with export, this made it visible in scope. Other suggestions?

Reply:
Resolved now (see https://github.com/stan-dev/projpred/pull/496#issuecomment-2229409972).